webpack.dev.js

```diff
@@ -10,7 +10,7 @@ module.exports = merge(common, {
 
   output: {
     filename: "[name].js",
-    chunkFilename: "[id].css"
+    chunkFilename: "[id].js"
```

Comment:
I believe this was a long standing copy/paste mistake.

Reply:
Does just changing this addresses the issue?

Reply:
Nope.